src/classes/CW_MarcomBriefViewSelectorControllerTest.cls

```diff
@@ -0,0 +1,82 @@
+/**
+* File Name   :    CW_MarcomBriefViewSelectorController.cls
+* Description :    B-109868: Add Related objects to the MarCom Brief View 
+*				   Displayes VF page for marcom and marketing users and 
+*				   standard page layout for other profiles
+*
+*
+* Modification Log
+* ============================================================================
+* Ver Date       Author         Modification
+* --- ---------- -------------- --------------------------
+* 0.1 05/06/2014 Cristina P.	Intial
+*
+*/
+@isTest
```

Comment:
Classes and triggers should use the `UpperCamelCase` naming convention. I suggest to rename accordingly.